packages/label/README.md

```diff
@@ -0,0 +1,25 @@
+# label
```

Comment:
This should be `@littlespoon/label`. Can you update the README?

Reply:
Updated